src/tools/crawling_tools.py

```
@@ -1038,3 +1234,104 @@ async def _update_sources(ctx: Context, source_content_map: Dict[str, str], sour
         except Exception as source_error:
             logger.warning(f"⚠️ Error updating source {source_id}: {source_error}")
 
+
+async def _ingest_pages_to_neo4j(ctx: Context, pages: list, source_type: str = "WEB_PAGE") -> int:
```

Ruff (0.12.2)

Unused function argument: `ctx`

(ARG001)